ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -14,7 +14,8 @@ import com.revenuecat.purchases.ui.revenuecatui.templates.Template2
 @Composable
 internal fun InternalPaywallView(
     offering: Offering? = null,
-    viewModel: PaywallViewModel = getPaywallViewModel(offering = offering),
+    listener: PaywallViewListener? = null,
+    viewModel: PaywallViewModel = getPaywallViewModel(offering = offering, listener = listener),
```

Comment:
@tonidero can you please check this out? I am not sure this approach is okay

Reply:
I think this looks ok. One option if we're worried about passing more parameters is to pass the entire `PaywallViewOptions` object instead of each of these separately.

Reply:
that might make sense yes... I think we can keep these three and if the number of parameters starts growing too much we can do that